ambry-api/src/test/java/com.github.ambry/rest/MockRestResponseChannel.java

```diff
@@ -245,6 +249,11 @@ private void setHeader(String headerName, Object headerValue, Event eventToFire)
           if (!responseMetadata.has(RESPONSE_HEADERS_KEY)) {
             responseMetadata.put(RESPONSE_HEADERS_KEY, new JSONObject());
           }
+          if (headerValue instanceof Date) {
```

Comment:
Not sure I understand this comment. 

This piece of code stores the `Date` instance as a converted `String` instance (as a real response channel would).